packages/styles/variables.css

```diff
@@ -31,6 +31,7 @@
   --accent-warning: #ffdd75;
   --accent-warning-light: #ffa1a1;
   --accent-caution: #ffe69f;
+  --accent-warning-dark: #b88a00;
```

Comment:
I do not feel strongly about the naming here, but it's currently named as "Pirate Gold" in UXPin, so figured I'd take a shot naming it.